tests/config/install.sh

```diff
@@ -41,6 +42,9 @@ ln -sf $SRC_PATH/strings_dictionary.xml $DEST_SERVER_PATH/
 ln -sf $SRC_PATH/decimals_dictionary.xml $DEST_SERVER_PATH/
 ln -sf $SRC_PATH/executable_dictionary.xml $DEST_SERVER_PATH/
 
+# custom TLD
+ln -sf $SRC_PATH/top_level_domains /var/lib/clickhouse/
```

Comment:
Should probably be `$DEST_SERVER_PATH` here. AFAIK fasttest uses `install.sh` with proper arguments, so it will work as well.

Reply:
Right and require path replace, already done